dragonfly/actions/action_mouse.py

```diff
@@ -215,8 +215,8 @@ def _process_window_position(self, spec, events):
     def _process_screen_position(self, spec, events):
         if not spec.startswith("[") or not spec.endswith("]"):
             return False
-        h_origin, h_value, v_origin, v_value = self._parse_position_pair(spec[1:-1])
-        event = MoveScreenEvent(h_origin, h_value, v_origin, v_value)
+        _, h_value, _, v_value = self._parse_position_pair(spec[1:-1])
```

Comment:
Should update the documentation at the top of the class :)

Reply:
Thanks! I've updated it now. How does it look?

Reply:
Looks good, thanks!

Reply:
Great!